frontend/src/app/modules/auth/register-form/register-form.component.html

```diff
@@ -0,0 +1,60 @@
+<div class="flex-container">
+  <div class="block-1">
+    <div class="registration-container">
```

Comment:
Please, make this part (.registration-container) as a separate component, as for login page it will look completely the same except form